### PR TITLE
mononoki: 1.3 -> 1.5

### PR DESCRIPTION
**Pull request by totoroot**
###### Description of changes

Update mononoki font from version 1.3 to 1.5

[Changes made with 1.4](https://github.com/madmalik/mononoki/tree/1.4)
[Changes made with 1.5](https://github.com/madmalik/mononoki/tree/1.5)

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/data/fonts/mononoki/default.nix [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
+{ lib, stdenv, fetchzip }:
 
-let
-  version = "1.3";
-in fetchzip {
-  name = "mononoki-${version}";
+stdenv.mkDerivation rec {
+  pname = "mononoki";
+  version = "1.5";
+  name = "${pname}-${version}";
```
Comment (SuperSandro2000) [inline pkgs/data/fonts/mononoki/default.nix:6]:
```suggestion
```

**Comment by totoroot** [comment]
Thanks, @SuperSandro2000! I've fixed mentioned points and pushed the updated changes.

File: pkgs/data/fonts/mononoki/default.nix [hunk 2]
```diff
@@ -1,22 +1,26 @@
...
 
 let
-  version = "1.3";
-in fetchzip {
-  name = "mononoki-${version}";
+  version = "1.5";
+  pname = "mononoki";
+in
 
+fetchzip {
```
[the remark this answers is not in the document]
Comment (totoroot) [reply pkgs/data/fonts/mononoki/default.nix]: Thanks, done!

**Comment by totoroot** [comment]
@SuperSandro2000 I've moved `pname` and `version` to `fetchzip` like you suggested.

**Comment by datMaffin** [comment]
Yesterday a commit was merged that makes another update to this PR necessary:

https://github.com/NixOS/nixpkgs/commit/f71d67800f55e7f4f4e00fda5bba3aa4d4184534#diff-65a0c27add0de775283f34c8579bbfa22f563dc45a8d66dc75fa5a47409622ef

**Pull request closed without merging**